modules/javafx.base/src/main/java/javafx/collections/transformation/TransformationList.java

```diff
@@ -91,7 +91,7 @@ public final boolean isInTransformationChain(ObservableList<?> list) {
         }
         List<?> currentSource = source;
         while(currentSource instanceof TransformationList) {
-            currentSource = ((TransformationList)currentSource).source;
+            currentSource = ((TransformationList<?, ?>)currentSource).source;
```

Comment:
Can use pattern matching for `instanceof`.
Also, can add a space after `while`.